### ISSUE TITLE
Doubt in vert_mask in run_skinning script

### ISSUE THREAD

**Issue by dhorka** [zhan-xu/RigNet#56]
Hi,

I have a couple of doubts related to how the loss is calculated. In specific in these lines:

```
   vert_mask = (torch.abs(skin_gt.sum(dim=1) - 1.0) < 1e-8).float()  #
   loss = cross_entropy_with_probs(skin_pred, skin_gt, reduction='none')
   loss = (loss * loss_mask_batch * vert_mask.unsqueeze(1)).sum() / (loss_mask_batch * vert_mask.unsqueeze(1)).sum()
```

I do not completly understand why is needed this vert_mask. The loss_mask_batch is the one used to filter the skinning_weights in the cases where k is greater than the number of bones of the current mesh, isn't it? But why is needed this vert_mask?

And the third line is not clear for me. I assume that you are trying to calculate the mean loss just for the real skinning weights, however, I do not see why you need to multiply for this vert_mask. 

Thanks!

**Comment by zhan-xu** [comment]
From what I can remember, vert_mask filters out "bad skinning vertices" where their skinning weights are not summed to 1. Suppose you have perfect training data, this mask will be all 1, which means it is of no use. But in case where you have some noise data, some vertices may have all-zero skinning weights. We don't want to involve those vertices to affect training.

**Comment by dhorka** [comment]
Thanks! I have miss this line.
